packages/xo-web/src/xo-app/sr/tab-disks.js

```diff
@@ -183,8 +355,28 @@ export default class SrDisks extends Component {
 
   render () {
     const vdis = this._getAllVdis()
+    const { newDisk } = this.state
+
     return (
       <Container>
+        <Row>
+          <Col className='text-xs-right'>
+            <TabButton
+              btnStyle={newDisk ? 'info' : 'primary'}
+              handler={this._toggleNewDisk}
+              icon='add'
+              labelId='vbdCreateDeviceButton'
+            />
+          </Col>
+          <Col>
```

Comment:
You can't put 2 `Col`s with `size` 12 (default size) in the same `Row`. Use 2 `Row`s instead.